src/child/index.js

```diff
@@ -20,6 +21,15 @@ require('script!../../node_modules/BitFlux/dist/bitflux.js');
 
 const store = configureStore();
 
+fin.desktop.main(() => {
+    fin.desktop.Window.getCurrent().addEventListener('bounds-changed', (e) => {
```

Comment:
I think it was mentioned offline that there are a few reasons for using `bounds-changed` over the alternative mentioned above. Please could the reason(s) be added here, so we have a record of this?